### ISSUE TITLE
Add history and log for admin operations on users profiles

### ISSUE THREAD

**Issue by Durafen** [iftechfoundation/ifdb-suggestion-tracker#178]
For now there is no way to see the history of a user (why got banned? When? ...)

**Comment by PatientRock** [comment]
I agree with this. We recently were contacted by a user who had been banned, but there was no discernable reason for why that had happened. The hope is that this community will continue to grow and as moderators come and go, there needs to be some record to consult when a past decision is up for review.

**Comment by dfabulich** [comment]
It occurred to me today that it's also not possible to view the history of user operations on users profiles; you can't see historical changes in a user's screen name, description, etc.

I think we'd still want to restrict viewing that history to admins only, so I think this is part of the same issue.
